File: StoryFlow_Shared/Models/GetStorySeasonDto.cs
namespace StoryFlow_Shared.Models
{
    public class GetStorySeasonDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public int SeasonNumber { get; set; }
        public bool IsVisibleForUser { get; set; }
        public List<GetStoryDto> Stories { get; set; } = new();
        public int? MaxPoints { get; set; }
    }
}

File: StoryFlow_Client/src/app/models/get-story-season-dto.ts
import { GetStoryViewDto } from "./get-story-view-dto";
import { GetWordLessonDto } from "./get-word-lesson-dto";

export interface GetStorySeasonDto {
    id: number;
    name: string;
    seasonNumber: number;
    isVisibleForUser: boolean;
    stories: GetStoryViewDto[];
    wordLessons: GetWordLessonDto[];
    maxPoints: number | null;
    pointsRequiredToUnlock: number | null;
}